app/controllers/pages_controller.rb

```diff
@@ -72,7 +70,20 @@ def get_page
 
   def request_country
     # when geocoder location API times out, request.location is blank
-    request.location.blank? ? nil : request.location.country_code
+    @request_country ||= request.location.try(:country_code)
```

Comment:
this is the wrong branch. you should have branched off `cache_invalidation` - this is no longer how this works